scripts/loom_to_h5ad.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import anndata as ad
+import loompy
+import os
+
+def convertloomtoh5ad(args):
+    """ Converts loom file format to h5ad. 
+
+    Args:
+        loom_path (str): path to loom file.  
+        output_path (str): output path directory to save h5ad file in. 
+    
+    """
+
+    file_name = os.path.basename(args.loom_file)
```

Comment:
this can be done all at once
```suggestion
    file_name = os.path.basename(args.loom_file).split(".")[0]
```

Reply:
Agreed. Thanks for the suggestion!